docs/Windows.md

```diff
@@ -1,43 +1,172 @@
+
+#### Swift Toolchain & Platform SDK concepts
+
+The installation instructions that follow will result in a directory tree that has a well thought-out structure. To understand this directory structure better you may consult  <https://github.com/compnerd/windows-swift/blob/master/docs/details.md>. But first, you might want to continue with the installation.
+
+#### Downloading the nightlies
+
+1. Go to  <https://dev.azure.com/compnerd/windows-swift>.
+2. Choose `Pipelines` > `Pipelines` from the left of the dashboard.
+3. Choose 'Runs'.
+4. Use the filter symbol to search for "VS2017".
+5. In the results list, scroll down until you see the first successful build (with a green OK symbol) and click on it.
+6. Under "Artifacts", click on the "... published" link.
+7. Download windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi by clicking on the appropriate down-arrows on the right. Be sure to really download these files from the same build (i.e. do not switch the build for the next download, and be careful when updating)! These files will be downloaded as zip files. Unless they are not automatically unzipped during the download process, unzip them to obtain *.msi files in the extracted directories.
+
+#### Installing the nightlies
+
+These *.msi files install the files to C:\Library. The complete Library directory can later be copied to a different location to be used there.
+
+Double-click windows-toolchain-amd64.msi, windows-sdk.msi, and windows-runtime-amd64.msi to run the according installations, one by one. (The installation process started by windows-toolchain-amd64.msi might take a while. Each of those installations can be deinstalled separately via the Windows "Apps & Features" settings. Such an deinstallation leaves all files in C:\Library that are not part of the according installation.)
+
+Those installations to not add any values to the PATH environment variable. The PATH environment variable has to adjusted separatly (see below).
+
+The directories created by those installations are as follows:
+
+- windows-toolchain-amd64.msi: `C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain`
+- windows-sdk.msi: `C:\Library\Developer\Platforms\Windows.platform`
+- windows-runtime-amd64.msi: `C:\Library\Swift\Current\bin`
+
+#### Further requirements: CMake
+
+On Windows, the most convenient setup for building Swift projects currently involves the use of CMake.  This requires CMake 3.15+ for Swift support.  You can download CMake from <https://cmake.org/>.
+
+In the following, we assume you added CMake to the C:\Libary folder, so that the CMake executable is in `C:\Library\cmake\bin`.
+
+#### Further requirements: ICU
+
+You will need the ICU libraries from <http://site.icu-project.org/>. The nightlies are built against ICU 64.2 from the ICU project. You can download the binaries for that from http://download./files/icu4c/64.2/icu4c-64_2-Win64-MSVC2017.zip.
```

Comment:
I think that this is a bad idea.  Different builds are currently using different ICUs.  See #8 which should hopefully pull everything to the locally built ICU 64.2.